index.html

```diff
@@ -0,0 +1,134 @@
+            <a class="nav-link" id="activ" aria-current="page" href="./about.html">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#programs">Program</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#performers">Courses</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#sponsors">Sponsors</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" id="imp" href="#sponsors"><button class="btn">US Community</button></a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+
+
+
+  <div class="content py-5 d-flex flex-column">
+    <p>"Hello! Sharing World!"</p>
+    <h1 class="pb-1 text-uppercase">JavaScript BootCamp <br> Global Summit 2023</h1>
+    <div class="card p-2">
+      <p>Javascript Media School Where Coding Dreams Comes True .</p>
+    </div>
+    <p class="text-uppercase text-dark">
+      <span class="fw-bold">2023.1.15(SUN),
+        5pm ~ 9pm (utc+8)</span> <br>
+
+      @Emei Mountain, USA
+    </p>
+  </div>
+
+  <section id="programs" class="my-2"></section>
+  <section id="performers" class="py-4"></section>
+
+  <section id="sponsors">
+    <div class="bg-grey">
+      <h2 class="text-center text-capitalize">Partner</h2>
+      <hr class="red">
+      <ul class= "d-flex gap-5 justify-content-center pt-4 text-grey">
+        <li>
+          <div class="thumbnail">
+            <a href="https://www.mozilla.org/en-US/foundation/" class="text-white text-muted fs-1 fw-bolder">mozilla</a>
+
+          </div>
+        </li>
+        <li>
+          <div class="thumbnail">
+            <a href="https://www.google.com/" class="text-grey fs-1 fw-light text-muted">Google</a>
+
+          </div>
+        </li>
+        <li>
+          <div class="thumbnail">
+            <a href="http://www.naver.com/" class="text-grey fs-1 text-muted">NAVER</a>
+          </div>
+        </li>
+        <li>
+          <div class="thumbnail">
+            <a href="http://www.kakaocorp.com/" class="text-white fs-1 text-muted">daumKakao</a>
+          </div>
+          <div class="thumbnail">
+            <a href="https://www.airbnb.co.kr/" class="text-grey fs-1 text-muted">airbnb</a>
+          </div>
+        </li>
+      </ul>
+    </div>
+  </section>
+
+  <footer class="container-fluid pt-2 light" id="about">
+    <div class="d-flex">
+      <a class="navbar-brand" href="./index.html"> <img class="img-fluid" id="log" src="./assets/images/java1.png"
+          alt="logo"><span class="fw-bold">JavaScript</span> </a>
+      <ul>
+        <li class="text-dark">
+          <h2> 2023 JavaScript Media <span>&copy;</span> Some Rights Reserved.</h2>
+        </li>
+
+      </ul>
+    </div>
+  </footer>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+  <script src="script.js"></script>
+</body>
+
+</html>
```

Comment:
I can see that most of the changes requested by the previous reviewers have not been implemented. Kindly note that it is required that the website is as identical to the [behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) as possible. You seem to be missing on this.

A discipline in following given designs as closely as possible would be a good great in preparation for a typical experience in your job after Microverse where you may be required to follow given designs as closely as possible.

Your design:
![image](https://user-images.githubusercontent.com/20554583/217743907-21a84bed-09c2-43ab-b11f-9c4abcbe929e.png)
![image](https://user-images.githubusercontent.com/20554583/217743937-6c287ea2-a633-4337-81dd-3ae027e56ee0.png)
![image](https://user-images.githubusercontent.com/20554583/217743957-aa8a86ad-53c1-4d56-89b1-b0e5aa0ebcaa.png)
![image](https://user-images.githubusercontent.com/20554583/217743973-d444925a-67b2-41fa-b4be-b24512f8918c.png)

Behance's Design:
![image](https://user-images.githubusercontent.com/20554583/217744481-d8825d75-122c-410a-a2b7-4c426706a669.png)
![image](https://user-images.githubusercontent.com/20554583/217744491-1e28b22f-795b-4817-8002-a7f314dfd009.png)
![image](https://user-images.githubusercontent.com/20554583/217744502-2c15499b-ffb0-4ab6-b19f-23af716166b4.png)
![image](https://user-images.githubusercontent.com/20554583/217744517-f3d8941b-2aea-4ca6-a48e-a5d177ebe24c.png)